StripeConnect/StripeConnect/Source/Components/ComponentType.swift

```diff
@@ -10,6 +10,7 @@ import Foundation
 /// The name of the embedded component tag in JS ([docs](https://docs.stripe.com/connect/supported-embedded-components))
 enum ComponentType: String, Encodable {
     /// Displays the balance summary, the payout schedule, and a list of payouts for the connected account
+    case accountManagement = "account-management"
     case payouts
```

Comment:
```suggestion
    case accountManagement = "account-management"
    /// Displays the balance summary, the payout schedule, and a list of payouts for the connected account
    case payouts
```

Reply:
oops, good catch. Probably got out of order when I rebased